scripts/ice-builder-xcode-wrapper

```diff
@@ -15,4 +15,14 @@ else
     FLAGS="--ice-home ../../../.."
 fi
 
-/usr/local/bin/icebuilder $FLAGS "$@"
+if [ -f "/usr/local/bin/icebuilder" ]; then
```

Comment:
Should we first check the path?

Reply:
How will you do that?

Reply:
Something like `if command -v icebuilder >/dev/null 2>&1; then` should work I think. Or we could maybe user `which ic builder`

Reply:
Ok I going to merge it as is, we can update it later if required